examples/security/keycloak-authorization/README.md

```diff
@@ -5,10 +5,10 @@ This folder contains an example `Kafka` custom resource configured for OAuth 2.0
 - the corresponding `oauth` authentication
 The folder also contains a Keycloak realm export to import into your Keycloak instance to support the example.
 
-Full instructions for the example are available in the [Strimzi Documentation](https://strimzi.io/docs/operators/0.24.0/using.html#con-oauth-authorization-keycloak-example).
+Full instructions for the example are available in the [Strimzi Documentation](https://strimzi.io/docs/operators/0.26.0/using.html#con-oauth-authorization-keycloak-example).
```

Comment:
I guess this could point to some downstream docs?

Reply:
[AMQ Streams on OpenShift documentation](https://access.redhat.com/documentation/en-us/red_hat_amq_streams/2.0/html-single/using_amq_streams_on_openshift/index?lb_target=stage#proc-oauth-authorization-keycloak-example_str)